src/core/xref.js

```diff
@@ -292,6 +292,7 @@ function addDataCiteToTerms(elems, queryKeys, data, conf) {
 
   for (let i = 0, l = elems.length; i < l; i++) {
     const elem = elems[i];
+    if (elem.closest("[data-no-xref]")) continue;
```

Comment:
Why not add this check here:
https://github.com/w3c/respec/blob/fc967beb562f59335877ed1190005080f5500d45/src/core/link-to-dfn.js#L45-L51

Reply:
I think this should work.
``` js
if (!foundDfn && linkTargets.length !== 0 && !("noXref" in ant.dataset)) { 
```